currency_rate_update/models/res_currency_rate_provider_ECB.py

```diff
@@ -44,12 +44,11 @@ def _obtain_rates(self, base_currency, currencies, date_from, date_to):
             return super()._obtain_rates(base_currency, currencies, date_from,
                                          date_to)  # pragma: no cover
 
-        # This provider only serves EUR-to-??? exchange rates
-        if base_currency != 'EUR':  # pragma: no cover
-            raise UserError(_(
-                'European Central Bank is suitable only for companies'
-                ' with EUR as base currency!'
-            ))
+        if base_currency != 'EUR' and \
+                base_currency in self._get_supported_currencies():
```

Comment:
Since you've added EUR to supported currencies list, you can just check for `base_currency != 'EUR'`